src/services/common/formatting.utils.ts

```diff
@@ -90,3 +94,71 @@ export function formatArea(value: number, digits = 2): string {
     return ''
   }
 }
+export function formatAddress(address: AddressResult | undefined) {
+  if (address === undefined) {
+    return i18next.t('N/A', { ns: 'client' })
+  }
+  return `${address.number}, ${address.street}, ${address.postal_code} ${address.locality}`
+}
+// export coordinates in different CRS as an 'xxx E | yyy N' pair
+// known targets:
+// EPSG:2169, EPSG:4326, EPSG:3263* (UTM zones 31N and 32N)
+// EPSG:4326:DMS (degree, minute, second), EPSG:4326:DMm (degree, minute.minute_decimals)
+export function formatCoords(
+  coords: Coordinate,
+  fromCrs: Projection | string,
+  format: string
+) {
+  const lonLanFormat = format.split(':')[2]
+  let toCrs = format
+  let utmZone = undefined
+  if (lonLanFormat !== undefined) {
+    toCrs = format.slice(0, format.lastIndexOf(':'))
+  } else if (format.endsWith('*')) {
+    const lonlat = toLonLat(coords, fromCrs)
+    utmZone = lonlat[0] <= 6 ? 'UTM31N' : 'UTM32N'
+    toCrs = format.replace('3*', utmZone.slice(3, 5))
+  }
+  const projectedCoords = transform(coords, fromCrs, toCrs)
+  const isDegrees = toCrs === 'EPSG:4326'
```

Comment:
use constants...

Reply:
I don't see much benefit, since the value of the constant will never change (projections are standard strings) but why not